src/lib/sticky-header/sticky-header.md

```diff
@@ -0,0 +1,48 @@
+Angular Material `<sticky-header>` is an attribute/component which makes DOM elements sticky to a specific parent.
+
+<!-- example(sticky-header-overview) -->
+
+The sticky-header is a useful component when users want to implement a container focusing on displaying headers,
```

Comment:
:+1: Great !!! Nice diagram :1st_place_medal: 
Changed the description and added the diagram~